### ISSUE TITLE
Latest release doesn't work, make npm release

### ISSUE THREAD

**Issue by smolagakuba** [rhys-vdw/ts-auto-guard#93]
running `ts-auto-guard` always produces the following error:

```
(node:14109) UnhandledPromiseRejectionWarning: TypeError: realpath is not a function
    at visitDirectory (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:18327:45)
    at Object.matchFiles (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:18323:13)
    at Object.matchFiles (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/ts-simple-ast/dist/typescript/tsInternal.js:7:35)
    at Object.readDirectory (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/ts-simple-ast/dist/utils/compiler/readDirectory.js:16:27)
    at Object.readDirectory (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/ts-simple-ast/dist/utils/tsconfig/getTsParseConfigHost.js:11:37)
    at getFileNamesFromConfigSpecs (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:29196:40)
    at matchFileNames (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:29116:16)
    at getFileNames (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:28665:26)
    at parseJsonConfigFileContentWorker (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:28597:18)
    at Object.parseJsonConfigFileContent (/home/kuba/.npm-global/lib/node_modules/ts-auto-guard/node_modules/typescript/lib/typescript.js:28544:16)
(node:14109) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag '--unhandled-rejections=strict' (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:14109) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

I tried to use the tool with and without arguments. Both result in the same error.

**Comment by Crusader4Christ** [comment]
npm version not bumped
so you can try
`npm i git+https://github.com/rhys-vdw/ts-auto-guard.git#57734a81829c8644b984bf2327bc54fc6340a5b8`
But for me it says just `Done!` and no files generated :(

**Comment by rhys-vdw** [comment]
Oh dear. Been a while since I tried it. Will take some looking into.

**Comment by rhys-vdw** [comment]
Even though this was fixed by #79, I will keep it open until I've cut a release.

**Comment by rhys-vdw** [comment]
Closed by #104
